### PR TITLE
Chore(deps): npm updates

### PR DESCRIPTION
**Pull request by johnrobeds**
## Description

- properties-file: 3.2.18 -> 3.2.19
- start-server-and-test: 1.14.0 -> 2.0.0
- node-mocks-http: 1.11.0 -> 1.13.0
- react-string-replace 1.1.0 -> 1.1.1
- @types/node: 17.0.45 -> 18.11.9
- @types/react: 18.0.25 -> 18.2.21
- @babel/core: 7.20.2 -> 7.22.15
- @wdio/browserstack-service: 8.16.0 -> 8.16.3
- @wdio/cli: 8.16.0 -> 8.16.3
- @wdio/local-runner: 8.16.0 -> 8.16.3
- @wdio/mocha-framework: 8.16.0 -> 8.16.3
- @wdio/spec-reporter: 8.15.10 -> 8.16.3

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -146,17 +146,17 @@
     "@testing-library/user-event": "^14.4.3",
     "@types/jest": "^29.5.4",
     "@types/js-cookie": "^3.0.3",
-    "@types/node": "^17.0.45",
-    "@types/react": "^18.0.25",
+    "@types/node": "^20.5.9",
```
Comment (jimCresswell) [inline package.json:149]: this should match the Node version we are using, so needs to be the latest 18.x version

File: package.json [hunk 2]
```diff
@@ -176,19 +176,19 @@
     "jest-environment-jsdom": "^29.6.2",
     "jest-fetch-mock": "^3.0.3",
     "json-diff": "^1.0.6",
-    "lint-staged": "^13.2.2",
-    "msw": "^1.2.5",
+    "lint-staged": "^14.0.1",
+    "msw": "^1.3.0",
```
Comment (jimCresswell) [inline package.json:180]: if this causes a test to fail you will need to catch up with @mantagen , he was looking at this

**Review by jimCresswell — APPROVED** [review]
Approving ahead of Node version getting lined up

File: package.json (hunk 1 above)
Comment (jimCresswell) [reply package.json:149]: I've approved ahead of the Node version getting lined up. If the change doesn't automatically dismiss this thread you should be able to click "Resolve conversation"

Comment (johnrobeds) [reply package.json]: Changed to 18.11.9

**Pull request merged**